en/mkdocs.yml

```diff
@@ -104,6 +104,7 @@ nav:
           - Develop a Scheduled Integration: develop-components/develop-integrations/develop-a-scheduled-integration.md
       - Develop a Web Application: develop-components/develop-a-web-application.md       
       - Develop Components With Git: develop-components/develop-components-with-git.md  
+      - Configuring mutual TLS between Components: develop-components/configuring-mutual-tls-between-components.md 
```

Comment:
Configure Mutual TLS Between Components

Reply:
Rename the file name to - configure-mutual-tls-between-components.md

Reply:
Updated with [cee6dc5](https://github.com/wso2/docs-choreo-dev/pull/942/commits/cee6dc53c13522052a972af5b1a9d1d36a359d04)